cryodrgn/commands/filter.py

```diff
@@ -37,9 +37,10 @@
 import matplotlib.pyplot as plt
 import seaborn as sns
 from matplotlib import colors
-from matplotlib.backend_bases import MouseButton, MouseEvent
-from matplotlib.widgets import LassoSelector, RadioButtons
+from matplotlib.backend_bases import Event, MouseButton, MouseEvent
```

Comment:
we use `MouseEvent` instead of `Event` for the type hints below for better specificity